### ISSUE TITLE
Summarization Examples: Support label_smoothed_cross_entropy

### ISSUE THREAD

**Issue by sshleifer** [huggingface/transformers#5168]
This seems to be used by nearly all papers, but the HF implementation in `examples/summarization/finetune.py` uses the standard cross entropy loss.

Ideally, we could expose a command line arg to allow this choice to be toggled, like: 

```python
parser.add_argument('--loss', type=str, choices=['cross_entropy', 'label_smoothed_cross_entropy'], default='cross_entropy')
```

**Comment by Laksh1997** [comment]
Yes, it's annoying that torch doesn't have an implementation of smoothed cross entropy.

Although note that using smoothed cross entropy will hurt the probability calibration of a seq2seq model (https://pubs.acs.org/doi/10.1021/acscentsci.9b00576)

**Comment by ieBoytsov** [comment]
@sshleifer This is good point. I would consider to work on this issue
As i see we need 2 things for this:
1) add an implementation of label_smoothed_cross_entropy
2) support choice of loss function in `finetune.py`

am i correct?

**Comment by sshleifer** [comment]
Yes you are!

**Comment by vblagoje** [comment]
This issue is outdated as label smoothing has already been integrated in master with https://github.com/huggingface/transformers/pull/5919
